tic-tac-toe/src/Square.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Square = (props) => {
+  return <button key={props.index} onClick={props.clickEvent} className='square'>{props.value}</button>;
```

Comment:
One tiny thing - the `key` prop should not be needed here. This is only important for React when you're rendering an array of items so it can keep track of which is which i.e. `{board.map(value => >Square key={value} />}` Basically, it's important to have it on the array side but not as needed here.